src/App.vue

```diff
@@ -6,73 +6,110 @@
             @click="createNewPlio"
             :isDisabled="pending"
           ></icon-button>
-        </div>
 
-        <div class="grid col-start-6 col-end-8 justify-items-end sm:col-start-7">
-          <!-- named routes - https://router.vuejs.org/guide/essentials/named-routes.html -->
+          <!-- home button -->
+          <icon-button
+            class="place-self-start"
+            :iconConfig="homeButtonIconConfig"
+            :titleConfig="homeButtonTextConfig"
+            :buttonClass="menuButtonsClass"
+            @click="redirectToHome"
+            :isDisabled="pending"
+          ></icon-button>
+
+          <!-- product guides -->
+          <icon-button
+            class="place-self-start"
+            :iconConfig="productGuidesButtonIconConfig"
+            :titleConfig="productGuidesButtonTextConfig"
+            :buttonClass="menuButtonsClass"
+            @click="redirectToProductGuides"
+            :isDisabled="pending"
+          ></icon-button>
+
+          <!-- docs -->
+          <icon-button
+            class="place-self-start"
+            :iconConfig="docsButtonIconConfig"
+            :titleConfig="docsButtonTextConfig"
+            :buttonClass="menuButtonsClass"
+            @click="redirectToDocs"
+            :isDisabled="pending"
+          ></icon-button>
+
+          <!-- whats new -->
+          <icon-button
+            class="place-self-start"
+            :iconConfig="whatsNewButtonIconConfig"
+            :titleConfig="whatsNewButtonTextConfig"
+            :buttonClass="menuButtonsClass"
+            @click="redirectToWhatsNew"
+            :isDisabled="pending"
+          ></icon-button>
+
           <!-- logout -->
-          <div v-if="showLogout" class="text-lg sm:text-xl">
-            <router-link v-if="!isAuthenticated" :to="{ name: 'Login' }">
-              <button
-                class="bg-white-500 hover:text-red-500 text-black font-bold border-0 object-contain"
-              >
-                {{ $t("nav.login") }}
-              </button>
-            </router-link>
-            <a href="#" v-if="isAuthenticated" @click="logoutButtonClicked">
-              <button
-                class="bg-white-500 hover:text-red-500 text-black font-bold border-0 object-contain px-1 py-2"
-              >
-                {{ $t("nav.logout") }}
-              </button>
-            </a>
-          </div>
-          <!-- locale switcher -->
-          <div class="self-center">
-            <LocaleSwitcher id="locale" class="flex justify-center"></LocaleSwitcher>
-          </div>
+          <icon-button
+            class="place-self-start"
+            :iconConfig="logoutButtonIconConfig"
+            :titleConfig="logoutButtonTextConfig"
+            :buttonClass="menuButtonsClass"
+            @click="logoutButtonClicked"
+            :isDisabled="pending"
+          ></icon-button>
         </div>
+        <router-view :class="routerViewClass" :key="$route.fullPath" />
```

Comment:
Why is the key required here?

Reply:
So, there was a bug before this: if you are in the editor and you click the create button from the menu, the URL changes but since the component doesn't change, it wouldn't reload the page.

Adding this lets Vue know that it should reload even if the component is the same.